buy_together_app/templates/buy_together_app/description.html

```diff
@@ -0,0 +1,30 @@
+{% load static %}
+<!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <title>{% block title %} buyTogether {% endblock %}</title>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+        <link rel="stylesheet" href="{% static 'css/website_description.css' %}">
+    </head>
+
+    <body>
+        {% include './parts/nav.html' %}
```

Comment:
I'm getting an error here: `TemplateDoesNotExist at /`
The given path is incorrect.

Reply:
For me it does work fine

Reply:
That doesn't make sense. there is no parts directory. Please check your path and files again.

Reply:
that is because parts was in gitignore,fixed!